CONTRIBUTING.md

```diff
@@ -4,22 +4,38 @@
 
 If you would like to contribute enhancements or fixes, please do the following:
 
-1.  Fork the package repository.
-2.  Run `npm install` to setup all dependencies.
-3.  Hack on a separate topic branch created from the latest `master`.
-4.  Check for lint errors with `npm run lint` or use `linter-eslint` within Atom.
-5.  Run the package specs.
-6.  Commit the changes and push the topic branch to your fork.
-7.  Make a pull request to the main repo.
-8.  Welcome to the club!
+1. Fork the package repository and clone your fork to local file.
+
+1. Run `npm install` in the project to setup all dependencies.
+
+1. Run `apm link --dev` to install your working folder as a development-mode-only package in Atom.
+
+1. While on the `master` branch run `git checkout -b my-super-special-branch` to create and checkout a separate branch for your work. This makes merging your changes a lot easier for the maintainers.
+
+1. Hack away at the code, and don't forget to have fun.
+
+1. Check for lint errors with `npm run lint` or use `linter-eslint` within Atom.
+
+1. Run `atom path/to/cool-project --dev` to open Atom in developer mode and test-drive your improvements.
+
+1. Run the package specs with `Ctrl-Y` or `⌘-Alt-Ctrl-P`, or by typing `specs` in the command pallete.
+
+1. Commit the changes and push the topic branch to your fork on GitHub.
+
+1. Make a pull request to the main repo [here](https://github.com/AtomLinter/linter-eslint/pulls).
+
+1. Welcome to the club! 🤸‍♀️
```

Comment:
Please use one of [these emoji](https://www.webpagefx.com/tools/emoji-cheat-sheet/), as GitHub has scripts in place to make them work on _any_ system while the Unicode emoji above only works in certain aspects.

(For example it's broken in source code view, but works in rendered view on my system.)

Reply:
`:man_carwheeling:` 🤸‍♂️  and `:woman_carthweeling:` 🤸‍♀️  both show in the dropdown when typing into GitHub comments. ??? Do you think it would make a difference if I used one of those instead of the actual emoji generated in the preview tab?

Reply:
Ah, exactly! I just assumed it wasn't in the list since you used the raw emoji instead of the text form. GitHub has some script in place where if the system doesn't understand the text form it will use an image, while on systems that understand it it uses the Unicode code point.

Reply:
You should _always_ use the text form wherever possible in Markdown documents as that allows _any_ text editor to work on the document, and systems without knowledge of the specific code point can still understand the document.